src/random_test.cpp

```diff
@@ -252,7 +252,7 @@ inline static std::string pick_algorithm_lock() {
   static auto lock = opt_string(LOCK);
   static auto algorithm = opt_string(ALGORITHM);
   std::string locks[] = {"DEFAULT", "EXCLUSIVE", "SHARED", "NONE"};
-  std::string algorithms[] = {"INPLACE", "COPY", "DEFAULT"};
```

Comment:
why you are removing default algorithms, It is valid syntax for most of ALTER. if you want you can add INSTANT.

Reply:
The reason for removing "default" is that the ALGORITHM default value is INPLACE which is already covered.

Reply:
no. better to retain it.  you can add one more if you want. but please verify it works for all combinations.